app/components/Roster.tsx

```diff
@@ -15,6 +15,12 @@ import styles from './Roster.css';
 
 const cx = classNames.bind(styles);
 
+const SignalStrength = {
```

Comment:
nit: don't need a variable since `None` and `High` are not being used. You can just use `50` in the classname.

Reply:
Done.